packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -359,7 +359,8 @@ class DropdownMenu<T> extends StatefulWidget {
   ///    virtual keyboard will not appear.
   ///
   ///  * On desktop platforms, acts as if set to true; the dropdown takes the
-  ///    focus when activated.
+  ///    focus when activated. This also enables keyboard navigation through
```

Comment:
I'm not sure we should add this comment as it seems to imply that keyboard navigation would be only available when this property is true (keyboard navigation is enabled as soon as the dropdown gets the focus, which can be achieved without using this property).

Reply:
I feel ambivalent about this one.

I can imagine someone being glad they saw this here; on the other hand, since focus can be obtained in a variety of ways, perhaps there's another spot where this comment would be more appropriate. (Maybe the top-level docs for the class declaration?)

Reply:
I've move the document to the top level, hope this make things clearer.